test/e2e/cloudstack_test.go

```diff
@@ -17,6 +17,46 @@ import (
 	"github.com/aws/eks-anywhere/test/framework"
 )
 
+// ApiServerExtraArgs
+func TestCloudStackKubernetes129RedHat8ApiServerExtraArgsSimpleFlow(t *testing.T) {
+	test := framework.NewClusterE2ETest(
+		t,
+		framework.NewCloudStack(t, framework.WithCloudStackRedhat129()),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube129)),
+		framework.WithClusterFiller(api.WithControlPlaneAPIServerExtraArgs("service-account-issuer", "https://my-custom-issuer-url")),
+		framework.WithClusterFiller(api.WithControlPlaneAPIServerExtraArgs("service-account-jwks-uri", "https://my-custom-issuer-url/openid/v1/jwks")),
+	)
+	runSimpleFlow(test)
+}
+
+func TestCloudStackKubernetes129Redhat8ApiServerExtraArgsUpgradeFlow(t *testing.T) {
+	var addApiServerExtraArgsclusterOpts []framework.ClusterE2ETestOpt
+	var removeApiServerExtraArgsclusterOpts []framework.ClusterE2ETestOpt
+	test := framework.NewClusterE2ETest(
+		t,
+		framework.NewCloudStack(t, framework.WithCloudStackRedhat129()),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube129)),
+	)
+	addApiServerExtraArgsclusterOpts = append(
+		addApiServerExtraArgsclusterOpts,
+		framework.WithClusterUpgrade(
+			api.WithControlPlaneAPIServerExtraArgs("service-account-issuer", "https://my-custom-issuer-url"),
+			api.WithControlPlaneAPIServerExtraArgs("service-account-jwks-uri", "https://my-custom-issuer-url/openid/v1/jwks"),
+		),
+	)
+	removeApiServerExtraArgsclusterOpts = append(
+		removeApiServerExtraArgsclusterOpts,
+		framework.WithClusterUpgrade(
+			api.RemoveAllAPIServerExtraArgs(),
+		),
+	)
+	runApiServerExtraArgsUpgradeFlow(
+		test,
+		addApiServerExtraArgsclusterOpts,
+		removeApiServerExtraArgsclusterOpts,
+	)
```

Comment:
Good design of specifying multiple upgrades 💯

Reply:
Thank you